app/src/main/java/com/example/happymeals/Recipe.java

```diff
@@ -42,11 +43,12 @@ public Recipe(){}
      * @param prepTime {@link Double} the time to prep the recipe measured in hrs.
      * @param servings {@link Double} The servings that the meal makes with the ingredients
      *                               described.
+     * @param type {@link String} the type of recipe (breakfast, lunch, etc)
      */
     public Recipe( double cookTime, String description, ArrayList< String > comments,
                    ArrayList< HashMap< String, DocumentReference > > ingredients,
                    ArrayList< String > instructions,
-                   double prepTime, double servings ) {
+                   double prepTime, double servings, String type ) {
```

Comment:
We do not need type inside the class as it will be stored under breakfast/lunch/dinner on a specific day. If you fetch it from breakfast you will already know it is breakfast.

Reply:
kk I'll remove it

Reply:
I got this mixed up with meal plan, but why do we need a type for recipe? Are we required to filter by breakfast/lunch/dinner??

Reply:
based on the storyboard I assumed it would be a good option to filter the recipes.